strawberry/http/parse_content_type.py

```diff
@@ -0,0 +1,17 @@
+from email.message import Message
```

Comment:
Maybe just call this module `parsers.py` to allow for more parsers in it in the future?